test/pool.test.ts

```diff
@@ -28,6 +28,20 @@ async function mineBlocksUntilCycleEnd(): Promise<void> {
 
 type AnyPool = EthPool | Erc20Pool;
 
+type ReceiverWeights = [PoolUser, number][];
+
+function receiverWeightsForContract(
+  weights: ReceiverWeights
+): {
+  receiver: string;
+  weight: number;
+}[] {
```

Comment:
Just a stylistic suggestion.
```suggestion
): Array<{
  receiver: string;
  weight: number;
}> {
```

For me, this is easier to parse since the important information is at the start of the type and not as easy to miss as `[]` at the end. I usually choose this syntax whenever it is more complicated than `SomeType[]`.

Reply:
I agree, fixed